Microsoft.Toolkit.Uwp.SampleApp/Package.appxmanifest

```diff
@@ -27,21 +27,22 @@
         <uap:SplashScreen Image="Assets\UWPCommunityToolkitSampleAppSplashScreen.png" BackgroundColor="#3750D1" />
       </uap:VisualElements>
       <Extensions>
-        <uap:Extension Category="windows.protocol">
-          <uap:Protocol Name="uwpct">
-            <uap:DisplayName>UWP Community Toolkit</uap:DisplayName>
-          </uap:Protocol>
-        </uap:Extension>
         <Extension Category="windows.backgroundTasks" EntryPoint="Microsoft.Toolkit.Uwp.Samples.BackgroundTasks.TestBackgroundTask">
           <BackgroundTasks>
             <Task Type="timer" />
           </BackgroundTasks>
         </Extension>
+        <uap:Extension Category="windows.protocol">
```

Comment:
Reason this moved down?

Reply:
It must have moved down when the I've created an appx for testing.